symbiont/main.py

```diff
@@ -50,9 +51,14 @@
 
 @app.get("/status")
 async def status_check():
-    return {"status": "up"}
+    status_response = {
+        "status": "up", 
+        "version": VERSION,
+        "environemnt": ENVIRONMENT,
+    }
+    return status_response
 
 
 @app.get("/")
 async def read_root():
-    return {"Hello": "World"}
+    return {"message": "Welcome to symbiont api! Check out the docs at /docs, frontend at https://symbiont.vercel.app and the source code on https://github.com/symbiont-me"}
```

Comment:
Could you move the string out to a variable please. This might need to be updated from time to time